src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -235,6 +235,10 @@ private void decodeOtherParameter(Position position, int id, ByteBuf buf, int le
             case 67:
                 position.set(Position.KEY_BATTERY, readValue(buf, length, false) * 0.001);
                 break;
+            case 68:
```

Comment:
Are you absolutely sure all the parameters are used by all Teltonika devices? Please provide documentation as a proof.

Reply:
Hello @tananaev, as i know teltonika opened they're information to public sources.. yes there may be compatibility problems with old and very old devices. Here is AVL procols:

https://wiki.teltonika-gps.com/view/FMB_AVL_ID
https://wiki.teltonika-gps.com/view/FMB640_AVL_ID

For CNG values, i just thought that it could be usable in feature for other devices.. don't know for now about that.

Reply:
The policy is to add only if multiple protocols already need it and it's a generic enough attribute. Only then we add it to a list of known attributes. Also, when you add it there, you should update the web app.

Otherwise I'm ok with adding custom attributes. I only have a problem with door info. That's too many attributes. I would rather keep it as a mask for now. You can always use computed attributes to decode the mask.

Reply:
ok, so do i need to cancel pull request and update ? :)

Reply:
You can update it.